elftools/dwarf/dwarfinfo.py

```diff
@@ -134,6 +137,9 @@ def __init__(self,
         self._cu_cache = []
         self._cu_offsets_map = []
 
+        # DWARF v4 type units by sig8 - OrderedDict created on Reference
```

Comment:
not sure what `on Reference` refers to here.

Do you mean to say it's lazily initialized the first time it's referenced?